index.html

```diff
@@ -19,8 +19,8 @@ <h2>Thank you for taking part.</h2>
       <label id="emiail-label"  for="email">Email</label>
       <input type="text" placeholder="Enter your email">
       <br>
-      <label id="age-label" for="number" >Age(optinal)</label><br>
-      <input type="number" min="10" max="99" placeholder="Age">
+      <label id="age-label" for="number" >Age</label><br>
```

Comment:
Hi, please make sure to add the placeholder for the word "Age" inside the label tag, to make it appear inside the input box instead of putting it between the label tag which will make it appear outside of the input box.